code/modules/mob/living/simple_animal/abnormality/he/singing_machine.dm

```diff
@@ -32,34 +39,52 @@ Finally, an abnormality that DOESN'T have to do any fancy movement shit. It's a
 
 	pixel_x = -8
 	base_pixel_x = -8
+	pixel_y = -6
+	base_pixel_y = -6
 	buckled_mobs = list()
 	buckle_lying = TRUE
 	max_boxes = 16
-	var/cleanliness = "clean"
-	var/statChecked = 0
+	var/statCheckPenalty = 0
 	var/bonusRed = 0
 	var/grindRed = 4
 	var/minceRed = 8
+	var/clampRed = 80
+	var/playWhite = 50
 	var/playTiming = 5 SECONDS
 	var/playLength = 60 SECONDS
-	var/playStatus = 0
-	var/playRange = 20
+	var/playStatus = SILENT
+	var/playRange = 15
 	var/noiseFactor = 2
+	var/list/musicalAddicts = list()
+
+	//Sound Vars
 	var/datum/looping_sound/singing_grinding/grindNoise
 	var/datum/looping_sound/singing_music/musicNoise
-	var/list/musicalAddicts = list()
+
+	//Visual Vars
+	var/obj/effect/singing_machine_hinge/machine_hinge
+	var/obj/particle_emitter/singing_note/particle_note
+	var/obj/particle_emitter/singing_note_broken/particle_note_broken
+	var/obj/particle_emitter/singing_grind/particle_grind
+	var/obj/particle_emitter/singing_blood/particle_blood
+
+/mob/living/simple_animal/hostile/abnormality/singing_machine/Initialize()
+	..()
+	machine_hinge = new/obj/effect/singing_machine_hinge(src)
```

Comment:
```suggestion
	machine_hinge = new(src)
```
Variable has it's type defined, unless you're making a subtype and treating it as the parent type then you don't need to list the type you're calling `new` on